pubspec.yaml

```diff
@@ -38,24 +38,26 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.3
 
+  dartx: any
```

Comment:
Add a comment for what we need this for, to be consistent with the other packages